dcrwallet.go

```diff
@@ -309,7 +305,14 @@ func rpcClientConnectLoop(legacyRPCServer *legacyrpc.Server, loader *wallet.Load
 			if legacyRPCServer != nil {
 				legacyRPCServer.SetChainServer(chainClient)
 			}
-			c <- chainClient
+			if cfg.EnableTicketBuyer {
+				loader.SetChainClient(chainClient.Client)
+				err := loader.StartTicketPurchase(nil, &cfg.tbCfg)
```

Comment:
this seems to be the only call to StartTicketPurchase so I would move the chain client into its parameter and then we don't need to deal with the possibility of forgetting to set it after a refactor later.